talend-component-maven-plugin/src/it/web/.jenkins/Jenkinsfile

```diff
@@ -152,141 +166,118 @@ pipeline {
+            runtimeVersion = "default"
           }
-          ///////////////////////////////////////////
-          // Print build config
-          ///////////////////////////////////////////
-          script {
 
-            println("Job info:\n" +
-                " - Tenant instance: ${tenantInstance}\n" +
-                " - Runtime version: ${runtimeVersion}\n" +
-                " - Test file executed: ${fileToRun}" + ".json\n" +
-                " - API Tester used environment: ${apiTesterEnv}\n")
-
-            println("Talend Test Orchestrator info:\n" +
-                "PLATFORM: ${params.PLATFORM}\n" +
-                "ENVIRONMENT: ${params.ENVIRONMENT}\n" +
-                "FRAGMENT_URL: ${params.FRAGMENT_URL}\n" +
-                "DEPLOYMENT_UNIT: ${params.DEPLOYMENT_UNIT}\n" +
-                "CYCLE: ${params.CYCLE}\n" +
-                "DEPLOYED_PRODUCTS: ${params.DEPLOYED_PRODUCTS}")
-
-            def runtimeName = "master"
-            if (runtimeVersion != "") {
-              runtimeName = runtimeVersion
-            }
-
-            // updating build displayName
-            currentBuild.displayName = (
-                "#" + currentBuild.number + ": " + fileToRun
-            )
-
-            // updating build description
-            currentBuild.description = (
-                "User: " + "${USER_NAME}" + " - " + "Runtime: " + runtimeName + "\n" +
-                    "Project: " + fileToRun + "\n"
-            )
+          println "Requested TCOMP_VERSION is ${runtimeVersion}\n"
+
+          if (runtimeVersion == "default") {
+            runtimeVersion = "${TCOMP_VERSION}"
+            println "RUNTIME_VERSION not supplied, local value will be used (${runtimeVersion})\n"
           }
 
-      }
-    }
-    stage('Checkout connector') {
-      steps {
-        // Checkout of the connector directory (TODO Will be cleaned when test component will be ready)
-        checkout([
-            $class           : 'GitSCM',
-            branches         : [[name: '*/master']],
-            extensions       : [
-                [$class: 'RelativeTargetDirectory', relativeTargetDir: './connectors'],
-                [$class: 'CleanBeforeCheckout', deleteUntrackedNestedRepositories: true]
-            ],
-            userRemoteConfigs: [[credentialsId: '"github-credentials"',
-                                 url: 'https://github.com/Talend/connectors-se.git']]
-        ])
+          // FILE_TO_RUN
+          try {
+            fileToRun = "${PROJECT_NAME}"
+          }
+          catch (groovy.lang.MissingPropertyException ignored) {
+            println "PROJECT_NAME not supplied, default value will be used (${fileToRun})\n"
+          }
+        }
+        ///////////////////////////////////////////
+        // Print build config
+        ///////////////////////////////////////////
+        script {
+          String user_name = currentBuild.getBuildCauses('hudson.model.Cause$UserIdCause').userId[0]
```

Comment:
Need a call about this part to understand.

Reply:
It is to get the Jenkins runner username from current Build